tests/test_pylast.py

```diff
@@ -1909,54 +1908,4 @@ def test_get_recent_tracks_from_to(self):
         self.assertEqual(str(tracks[0].track.artist), "Johnny Cash")
         self.assertEqual(str(tracks[0].track.title), "Ring of Fire")
 
-
-if __name__ == '__main__':
```

Comment:
I'd like to keep this main section, it's useful for running specific tests of the slow, integration tests. I know you can target from py.test, but `tox -e py27 -- project_name_here/tests/folder_name1_here/folder_name2_here/test_file_name_here.py:TestClassName.test_method_name` is a bit unwieldy.